src/Fixer/Symfony/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -43,6 +54,7 @@
      * - 'return' remove blank lines after a line with a 'return' statement
      * - 'throw' remove blank lines after a line with a 'throw' statement
      * - 'use' remove blank lines between 'use' import statements
+     * - 'curly_brace_open' remove blank lines after a curly opening brace ('{')
```

Comment:
This option is so powerful !
Actually, you could now remove `NoBlankLinesAfterClassOpeningFixer` and move all it's tests here, tested with `curly_brace_open`. It works!